File: json_data_extraction.py
import requests

# GitHub Raw URL
# kendi json formatındaki verimi alıyorum
raw_url = "https://raw.githubusercontent.com/emredeveloper/Database/main/db.json"

# API İsteği Gönderme
response = requests.get(raw_url)

# Yanıtı Kontrol Etme
if response.status_code == 200:
    # gelen veriyi json formatında döndürüyorum adettendir
    data = response.json()

    # Alınan Veriyi Gösterme
    print("Alınan Veri:")
    print(data)

else:
    print(f"Hata Kodu: {response.status_code}")
    print(f"Hata Mesajı: {response.text}")

File: json_data_extraction.js
// Node.js ortamında çalışacaksa 'node-fetch' modülünü yükleyin:
// npm install node-fetch
const fetch = require('node-fetch');

// GitHub Raw URL
const rawUrl = "https://raw.githubusercontent.com/emredeveloper/Database/main/db.json";

// API İsteği Gönderme
fetch(rawUrl)
  .then(response => {
    if (!response.ok) {
      throw new Error(`HTTP hata! Hata kodu: ${response.status}`);
    }
    return response.json();
  })
  .then(data => {
    // Alınan Veriyi Gösterme
    console.log("Alınan Veri:");
    console.log(data);
  })
  .catch(error => {
    console.error("Hata:", error.message);
  });
